main.py

```diff
@@ -7,35 +7,42 @@
 
 collections.Callable = collections.abc.Callable
 
-print("---------------------------------------------------------------------------------")
-print("Welcome in app Altalite!\n"
-      "Enter below the ticker of the company you are interested in and I will analyze it for you")
-print("---------------------------------------------------------------------------------")
+CompanyTicker = input("Enter company ticker you want analyze:")
```

Comment:
To można by lepiej nazwać, żeby dalej w kodzie było wiadomo skąd ten ticker się bierze, więc np. `CompanyTickerFromUserInput()` czy jakoś tak

Reply:
Poprawione dodałem commita